src/components/GoToAllRestaurantsButton/GoToAllRestaurantsButton.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from "./GoToAllRestaurantsButton.module.scss";
+import goToIcon from "../../assets/images/goToIcon.svg";
+import { Link } from "react-router-dom";
+
+const GoToAllRestaurantsButton = () => {
+  return (
+    <Link to="/restaurants" className={styles.linkContainer}>
```

Comment:
What do you think could be the problem with this link?

Reply:
@nitzanpap  done